<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRImagingFCSSeriesDialog</class>
 <widget class="QDialog" name="QFRDRImagingFCSSeriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;parameter:</string>
       </property>
       <property name="buddy">
        <cstring>cmbPrameter</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbPrameter">
       <item>
        <property name="text">
         <string>binning</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>start/stop/delta</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;start:</string>
         </property>
         <property name="buddy">
          <cstring>edtStart</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="JKDoubleEdit" name="edtStart">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>st&amp;op:</string>
         </property>
         <property name="buddy">
          <cstring>edtStop</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="JKDoubleEdit" name="edtStop">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="JKDoubleEdit" name="edtDelta">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&amp;delta:</string>
         </property>
         <property name="buddy">
          <cstring>edtDelta</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>jkdoubleedit.h</header>
   <slots>
    <signal>valueChanged(double)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbPrameter</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtStart</tabstop>
  <tabstop>edtStop</tabstop>
  <tabstop>edtDelta</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFRDRImagingFCSSeriesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFRDRImagingFCSSeriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
